vhdbuilder/packer/install-dependencies.sh

```diff
@@ -296,8 +296,8 @@ AMD64_ONLY_CNI_VERSIONS="
 "
 #Please add new version (>=1.4.12) in this section in order that it can be pulled by both AMD64/ARM64 vhd
 MULTI_ARCH_VNET_CNI_VERSIONS="
-1.4.22
```

Comment:
azure cni binaries of 1.4.22 is used by aks-rp for now, we still need to cache it in VHD